crates/polars-ops/src/series/ops/is_first.rs

```diff
@@ -37,11 +36,53 @@ fn is_first_bin(ca: &BinaryChunked) -> BooleanChunked {
 fn is_first_boolean(ca: &BooleanChunked) -> BooleanChunked {
     let mut out = MutableBitmap::with_capacity(ca.len());
     out.extend_constant(ca.len(), false);
-    if let Some(index) = arg_max_bool(ca) {
```

Comment:
I believe the original code will produce incorrect results, so fix it by the way. But I am not sure if the following implementation can be simplified. 🤔

Reply:
Ah, it ignored the `false` and `null` you mean?